CHANGELOG.asciidoc

```diff
@@ -34,6 +34,10 @@ Use subheadings with the "=====" level for adding notes for unreleased changes:
 [[release-notes-1.x]]
 === Java Agent version 1.x
 
+[float]
```

Comment:
This is the wrong place, it must be above `[[release-notes-1.x]]` but under `===Unreleased`